### ISSUE TITLE
Initial boot

### ISSUE THREAD

**Issue by CatboyCoding** [ivanmeler/android_kernel_samsung_herolte#12]
I've been stuck on the loading screen for 20 minutes, how long does it usually take?

**Comment by CadeusTheGreat** [comment]
When I first booted my phone with this ROM it took a good 20-25mins. Just need to be very patient with it is all! :) Few more minutes at most and it should be up!

**Comment by jackhistoria** [comment]
> When I first booted my phone with this ROM it took a good 20-25mins. Just need to be very patient with it is all! :) Few more minutes at most and it should be up!

2h and it is still starting up after sideloading the latest update. Did it work for you after half an hour @LilSizzurp? (Never had any problem with an update in the last 2 years)

**Comment by ivanmeler** [comment]
> > When I first booted my phone with this ROM it took a good 20-25mins. Just need to be very patient with it is all! :) Few more minutes at most and it should be up!
> 
> 2h and it is still starting up after sideloading the latest update. Did it work for you after half an hour @LilSizzurp? (Never had any problem with an update in the last 2 years)

did you by any chance install twrp manager app? because that can cause issues, also did you update gapps because they also had some faulty releases (opengapps team)

**Comment by jackhistoria** [comment]
> > > When I first booted my phone with this ROM it took a good 20-25mins. Just need to be very patient with it is all! :) Few more minutes at most and it should be up!
> > 
> > 
> > 2h and it is still starting up after sideloading the latest update. Did it work for you after half an hour @LilSizzurp? (Never had any problem with an update in the last 2 years)
> 
> did you by any chance install twrp manager app? because that can cause issues, also did you update gapps because they also had some faulty releases (opengapps team)

Yes, the checkmark was set for installing the twrp app afterwards by default, guess it is that :( 
Any way to fix it? I will try to sideload both again, the os update and the gapps update unchecking the twrp afterwards. Thanks for your fast response and your great work over the years, (again) really appreciate it.

EDIT:
Yes that was it! First the latest update `lineage-16.0-20200712-UNOFFICIAL-herolte` (clean the cache activated), directly OpenGapps afterwards, reboot - ~7 minutes - boom. Thank god! (`std::string god =  "Ivan Meler"`)
